armi/reactor/blocks.py

```diff
@@ -1788,6 +1807,11 @@ def getMaxArea(self):
         .. impl:: Area of block is retrievable.
             :id: I_ARMI_BLOCK_DIMS2
             :implements: R_ARMI_BLOCK_DIMS
+
+            This method first retrives the pitch of the Hexagonal Block
+            (:need:`I_ARMI_UTIL_HEXAGON0`) and then leverages the 
+            area calculation via :need:`I_ARMI_UTIL_HEXAGON0`.
```

Comment:
Is the first :need:`I_ARMI_UTIL_HEXAGON0` extra?

Reply:
Its intent is to point to the definition of the hex Block. 
https://github.com/terrapower/armi/pull/1619/commits/efac243a8320f6f8e44e6f6c6177aae96054fb2d